### ISSUE TITLE
Could you publish the loss curves for both pretraining and finetuning?

### ISSUE THREAD

**Issue by hill2hill** [DAMO-NLP-SG/VideoLLaMA2#49]
Thank you for your excellent work. I am attempting to replicate the pretraining and lora finetuning processes, and I would like to confirm whether my model's convergence is reasonable.

**Comment by LiangMeng89** [comment]
Hello,I'm a phD student from ZJU, I also use videollama2 to do my own research,we create a WeChat group to discuss some issues of videollama2 and help each other,could you join us? Please contact me: WeChat number == LiangMeng19357260600, phone number == +86 19357260600,e-mail == liangmeng89@zju.edu.cn.
